docs/high-performance-order-storage/guide-large-store.md

```diff
@@ -0,0 +1,134 @@
+
+## Before you begin: Get familiar with the HPOS project and related documentation
+
+1. [High-Performance Order Storage](https://woocommerce.com/document/high-performance-order-storage/)
+2. [High-Performance Order Storage Upgrade Recipe Book](https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book)
+3. [High-Performance Order Storage: Backward Compatibility and Synchronization](https://developer.woocommerce.com/2022/09/29/high-performance-order-storage-backward-compatibility-and-synchronization/)
+
+## Phase 1: Test out HPOS on a local development system
+
+If you have a local development setup for your site, you can try out HPOS on your development setup. Make sure that you have all plugins and any custom code enabled on this test site and that you’re running their latest versions. The test environment should mimic your production site as much as possible.
+
+At the very least, we suggest you focus on the following tests:
+
+1. Test checkout with every payment method.
+2. Test refunds for those checkouts.
+3. If you are using WooCommerce Subscriptions, check that subscriptions can be purchased and renewals come through.
+4. Any critical flows you’ve defined for your site.
+
+You can repeat the tests with both synchronization enabled and disabled.
+
+Remember that you can [toggle synchronization](https://woocommerce.com/document/high-performance-order-storage/#section-5) by going to **WooCommerce > Settings > Advanced > Features**, and toggling the “**Enable compatibility mode**” checkbox.
+
+![Toggle synchronization](https://woo.com/wp-content/uploads/2023/10/image-18.png)
+
+**Review: Phase 1 Checklist**
+
+- [ ] Make sure code on your test site is up to date by grabbing the latest versions of all WooCommerce-related plugins you’re running.
+- [ ] Test checkout functionality for every payment method.
+- [ ] Verify refund process for test checkouts.
+- [ ] If WooCommerce Subscriptions is installed, test subscription purchases and renewal.
+- [ ] Test critical site flows with synchronization both enabled and disabled.
+
+## Phase 2: Migrations and testing on a staging site
+
+When you’re happy with how things are working in your local environment, it is time to copy over your production database to the staging site and test out HPOS migrations. For large sites, we recommend setting synchronization to on but, instead of relying on the scheduled jobs to perform the migration, you can speed things up by using the CLI: specifically, the command **wp wc cot sync**.
+
+To have a sense of how much the final migration (on production) is going to take, make sure to time how much it takes for this command to run. For example, for our test store of 9 million orders, this took about a week to complete.
```

Comment:
To `get` a sense of how much `time` the final migration (on production) is going to take